one_fm/patches.txt

```diff
@@ -124,3 +124,4 @@ one_fm.patches.v15_0.add_update_custom_field #2025-05-20
 one_fm.patches.v15_0.add_update_property_setter #2025-05-20
 one_fm.patches.v15_0.add_update_assignment_rule #2025-05-20
 one_fm.patches.v15_0.add_update_workflow #2025-06-04
+one_fm.patches.v15_0.add_site_project_manager 
```

Comment:
No need to add new patch use the existing patch `one_fm.patches.v15.add_update_<doctype>_custom_field`